targetframework.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+    <PropertyGroup Condition="! $(OS.Contains('win')) AND '$(TargetFramework)'== 'net461'">
+        <FrameworkPathOverride>$(NuGetPackageRoot)microsoft.targetingpack.netframework.v4.6.1/1.0.1/lib/net461/</FrameworkPathOverride>
+        <RestoreAdditionalProjectSources>https://dotnet.myget.org/F/dotnet-core/api/v3/index.json</RestoreAdditionalProjectSources>
+    </PropertyGroup>
+    <ItemGroup Condition="! $(OS.Contains('win')) AND '$(TargetFramework)'== 'net461'">
+        <PackageReference Include="Microsoft.TargetingPack.NETFramework.v4.6.1"  Version="1.0.1" ExcludeAssets="All" PrivateAssets="All" />
```

Comment:
Should this package version number live in `versions.props`?

Reply:
Makes sense .. let me do that.

Reply:
So the options are either move the whole thing into versions.props or make this dependent on version.props and remove the versions.props dependency in the rest of the project files since it will come through from targetframework.props. both seem bad to me.